src/components/Admin/Toolbar.jsx

```diff
@@ -4,9 +4,34 @@ import Checkbox from "../Checkbox";
 import { HiSearch } from "react-icons/hi";
 import Tag from "./Tag.jsx";
 import { FaDownload, FaTrashAlt } from "react-icons/fa";
+import { CSVLink } from "react-csv";
 import { colors } from "@/data/Tags";
 
-const Toolbar = ({ input, setInput, tags, setObjects, objects, filters }) => {
+function convert(input) {
+  if (Array.isArray(input)) {
+    return input.map(convert).join(", ");
+  } else if (typeof input === "boolean") {
+    return input ? "True" : "False";
+  } else if (typeof input === "object") {
+    return Object.entries(input)
+      .map(([key, value]) => `${key}: ${convert(value)}`)
+      .join(", ");
+  } else if (typeof input === "number") {
+    return input.toString();
```

Comment:
do we need to convert to a string? does it not work when we have a number?

Reply:
It was just in case we needed it. According to my comment above with the screenshots, it seems like we don't need it

Reply:
remove the toString then, if we dont need it